Sources/BraveWallet/WalletConstants.swift

```diff
@@ -97,6 +97,6 @@ struct WalletConstants {
 }
 
 public struct WalletDebugFlags {
-  public static let isSolanaDappsEnabled: Bool = false
+  public static let isSolanaDappsEnabled: Bool = true
```

Comment:
Fine until approved, but we should restore this to false before merging. We'll remove this flag and enable Solana Dapps by default in https://github.com/brave/brave-ios/issues/6154 with full Solana Dapps security review after this PR merges 🙂